playground/test-utils.ts

```diff
@@ -71,7 +71,12 @@ const timeout = (n: number) => new Promise((r) => setTimeout(r, n))
 
 async function toEl(el: string | ElementHandle): Promise<ElementHandle> {
   if (typeof el === 'string') {
-    return await page.$(el)
+    const realEl = await page.$(el)
+    if (realEl == null) {
+      throw new Error(`can not find element: "${el}"`)
+    }
+
+    return realEl
```

Comment:
```suggestion
    const realEl = await page.$(el)
    if (realEl == null) {
      throw new Error(`Cannot find element: "${el}"`)
    }
    return realEl
```

Tiny nit, but this change makes sense to me!